### PR TITLE
Fix download for v. 3.9

### PR DESCRIPTION
**Pull request by dfev77**
Also:
 - increase default nunit runner to 2.9.0
 - only take assemblies matching regex /test(s?).dll/

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,10 +4,15 @@
 * upgrade gradle-download-plugin to 3.2
 * upgrade net.researchgate.release to 2.6.0
 * built with gradle 4.2.1
+* increase default NUnit runner version from 2.6.4 to 3.9.9
```
Comment (ngyukman) [inline CHANGELOG.md]: it has to be in new version, changes before has been released

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,23 @@
 # gradle-nunit-plugin changelog
+##1.13
```
Comment (ngyukman) [inline CHANGELOG.md:2]: miss ###changed?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,11 +1,23 @@
...
+* increase default NUnit runner version from 2.6.4 to 3.9.0
+* take as test assemblies only those containing "test.dll" or "tests.dll"
+to not require explicit config for projects like "test-framework.dll"
+* built with gradle 4.10.2
+
+### Fixed
+* Apply default parameters on plugin `com.ullink.msbuild` instead of `msbuild`
```
Comment (ngyukman) [inline CHANGELOG.md]: this was an item in 1.12

File: src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy [hunk 4]
```diff
@@ -86,6 +91,10 @@ class NUnit extends ConventionTask {
         if (getIsV3(version)) {
             this.metaClass.mixin NUnit3Mixins
         }
+        else
+        {
```
Comment (ngyukman) [inline src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy]: should follow the same code convention, having curly bracket on the previous line

File: src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy [hunk 5]
```diff
@@ -134,9 +143,13 @@ class NUnit extends ConventionTask {
     }
 
     String getFixedDownloadVersion() {
-        def version = getNunitVersion()
+        String version = getNunitVersion()
         if (isV35OrAbove && version.endsWith('.0')) {
-            return version.take(version.length() - 2)
+            version = version.take(version.length() - 2)
+            if (isV39)
+            {
```
Comment (ngyukman) [inline src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy]: here as well

Comment (dfev77) [reply src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy]: done

File: CHANGELOG.md (hunk 2 above)
Comment (dfev77) [reply CHANGELOG.md:2]: done

File: CHANGELOG.md (hunk 3 above)
Comment (dfev77) [reply CHANGELOG.md]: my bad, fixed

File: src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy (hunk 4 above)
Comment (dfev77) [reply src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy]: done

**Review by ngyukman — APPROVED** [review]

**Pull request merged**